src/transfers-history/services/SpokePoolEventsQueryService.ts

```diff
@@ -39,17 +39,27 @@ export class SpokePoolEventsQueryService {
         blockTimestampMap: {},
       };
     }
-
-    [depositEvents, filledRelayEvents] = await Promise.all([
+    let start = new Date();
+    const [depositEventsSettled, filledRelayEventsSettled] = await Promise.allSettled([
       this.eventsQuerier.getFundsDepositEvents(from, to, this.depositorAddr),
       this.eventsQuerier.getFilledRelayEvents(from, to, this.depositorAddr),
     ]);
-
+    depositEvents = depositEventsSettled.status === "fulfilled" ? depositEventsSettled.value : [];
+    filledRelayEvents = filledRelayEventsSettled.status === "fulfilled" ? filledRelayEventsSettled.value : [];
```

Comment:
Was this the bug? If so, what did the allSettled fix?

Reply:
The issues were in the [repository file](https://github.com/across-protocol/sdk-v2/pull/27/files/39e2ebd8048b109503205ceca909a577ec56d9be#diff-f833d41d83e455eb24eb6451414108f4d4786a3e59d699f33585de218415aade)

`allSettled` is just to make sure that throwing errors does not prevent getting events which might be fetched successfully